TimeFormat.java

```diff
@@ -12,6 +12,47 @@ public static void main(String[] args) {
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+		if (hours>12) {

```

Comment:
This code has repeated code. this code is harder to maintain. 
consider the following:
```
String minutesStr = "";
String hoursStr = "";
String amPmStr = "";
if (minutes >= 10) {
   minutesStr += minutes;
} else {
   minutesStr += "0" + minutes;
}

if (hours <= 12) {
    hoursStr += hours; 
} else {
    hoursStr += (hours - 12); 
}

if (hours < 12){
   amPmStr += "AM";   
} else {
   amPmStr += "PM";
}

System.out.println(hoursStr + ":" + minutesStr + " " + amPmStr);
```